dist/t/spec/features/0010_authentification_spec.rb

```diff
@@ -1,17 +1,22 @@
 require "spec_helper"
 
-RSpec.describe "Sign Up & Login" do
-  it "should be able to sign up successfully and logout" do
+RSpec.describe "Authentification" do
```

Comment:
Nope! According to [Wiktionary](https://en.wiktionary.org/wiki/authentification) it is a misspelling of Authentication (from German Authentifikation). See also [stackexchange](http://english.stackexchange.com/questions/7844/is-authentification-a-real-word)